lib/lotus/logger.rb

```diff
@@ -0,0 +1,88 @@
+require 'logger'
+
+module Lotus
+  # Lotus default logger
+  #
+  # Implement with the same interface of ruby std ::Logger
+  # Optout the logdev parameter and always use STDOUT to output the log
+  #
+  # Lotus logger also has the app_tag concept, which used to identify
+  # which application the log come from
+  #
+  # Example Usage:
+  # #1 Logger with namespace
+  # module App
+  #   class AppLogger < Lotus::Logger
+  #   end
+  #
+  #   def foo
+  #     Applogger.new.info('foo')
+  #     #=> output: I, [2015-01-10T21:55:12.727259 #80487]  INFO -- [App] : foo
+  #   end
+  # end
+  #
+  # #2 Logger without namespace
+  # class AppLogger < Lotus::Logger
+  # end
+  #
+  # Applogger.new.info('foo')
+  # #=> output: I, [2015-01-10T21:55:12.727259 #80487]  INFO -- [Shared] : foo
+  #
+  # @since 0.1.0
```

Comment:
`since` is for version of lotus that introduces this change, please leave it as `x.x.x`